Documentation/nuget-dependencies-for-the-toolchain.md

```diff
@@ -1,27 +0,0 @@
-#NuGet dependencies to use ILC
-
```

Comment:
If there is a good reason why we need to have this in documentation, I can put add it back.